src/libraries/Common/src/ShouldSkipLocalsInit.cs

```diff
@@ -0,0 +1,7 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// Used to indicate to the compiler that the .locals init flag should
+// not be set in nested method headers when emitting to metadata.
```

Comment:
```suggestion
// not be set in method headers.
```
- "nested" does make sense here. Nested into what?
- Method headers are not metadata

Reply:
Lifted from https://github.com/dotnet/runtime/blob/a021db003acfaebd13faaa6e79c032bc11c921ff/src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/SkipLocalsInitAttribute.cs#L8-L9. I'll update the documentation there as well